### ISSUE TITLE
[Accessibility]: 

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#43272]
### Audit type

Screen reader navigation

### Problem description

Insights data points do not announce X/Y values, only say "Click to see data point details" (which is also not very precise, since it links to a diff search).

Uploading Screen Recording 2022-10-20 at 17.56.50.mov…

### Expected behavior

The data values need to announce the X and Y values of the data point (i.e. provide a programmatically associated text alternative to the non-text presentation of the values).

### Additional details

[1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by vovakulikov** [comment]
Duplicate of https://github.com/sourcegraph/sourcegraph/issues/41688
